drivers/flash/spi_flash_at45.c

```diff
@@ -681,6 +699,10 @@ static const struct flash_driver_api spi_flash_at45_api = {
 			.cs_gpio = DT_INST_SPI_DEV_CS_GPIOS_LABEL(idx),      \
 			.cs_pin  = DT_INST_SPI_DEV_CS_GPIOS_PIN(idx),	     \
 			.cs_dt_flags = DT_INST_SPI_DEV_CS_GPIOS_FLAGS(idx),)) \
+		IF_ENABLED(DT_INST_NODE_HAS_PROP(idx, reset_gpios), (		     \
+			.reset_gpio = DT_INST_GPIO_LABEL(idx, reset_gpios),      \
```

Comment:
```suggestion
			.reset_gpio = DEVICE_DT_GET(DT_GPIO_CTLR(idx, reset_gpios)),      \
```

Reply:
@mbolivar-nordic 
Sure, it's a good reason. Should I change other calls in this driver as well to keep code uniform?

Reply:
> Sure, it's a good reason. Should I change other calls in this driver as well to keep code uniform?

@Delsian that would be great, thank you. Since this is a SPI device, you can try to use SPI_CONFIG_DT_INST to get its SPI configuration without so much boilerplate code. See drivers/sensor/bme280/bme280.c for example usage.

Reply:
Well, I thought it better to keep commits atomic, so I did only change in reset pin and left SPI driver call untouched and ready for next change.